css/style.css

```diff
@@ -0,0 +1,1122 @@
+}
+
+.fix-me {
+  z-index: 99;
+}
+
+.header-title-2 {
+  position: relative;
+  height: 60%;
+  width: 100%;
+  background: linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0) 100%);
+  z-index: 999;
+}
+
+[class*='header-title'] h2 {
+  font-family: 'Rajdhani', sans-serif;
+  font-size: 14px;
+  color: white;
+  margin: 0;
+}
+
+.overlay-container {
+  width: 100%;
+  height: 100%;
+  padding: 0 20px;
+  box-sizing: border-box;
+}
+
+.overlay-title {
+  font-size: 1.5em;
+  color: white;
+  font-family: 'Ramabhadra', sans-serif;
+  line-height: 25px;
+}
+
+.overlay-title-1 {
+  font-size: 2.5em;
+  color: white;
+  font-family: 'Ramabhadra', sans-serif;
+  line-height: 50px;
+}
+
+.overlay-subtitle {
+  font-size: 12px;
+  color: gray;
+}
+
+nav {
+  width: 100%;
+  height: 80px;
+  background-color: white;
+  display: float;
+  position: fixed;
+  z-index: 9999;
+  margin-top: -110px;
+}
+
+.nav-bar-1 {
+  width: 100%;
+  margin: 0;
+  height: 28px;
+  background-color: black;
+  display: inline-block;
+}
+
+.nav-bar-2 {
+  width: 100%;
+  margin: 0;
+  height: 52px;
+  background-color: #fff;
+}
+
+.tnw-logo-container {
+  float: left;
+  height: 80px;
+  width: 13%;
+  margin: 0;
+  background: linear-gradient(168deg, rgba(204, 0, 153, 1) 0%, rgba(240, 48, 69, 1) 100%);
+  display: none;
+}
+
+.nav-bar-1 a {
+  margin: auto 10px;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+}
+
+.float-left {
+  float: left;
+  width: auto;
+  margin-left: 30px;
+}
+
+.float-right {
+  float: right;
+  width: auto;
+  margin-left: 16px;
+}
+
+.yellow-thingy {
+  width: 100%;
+  height: 30px;
+  margin-top: 0;
+  margin-bottom: 3px;
+  background-color: #f4d03f;
+}
+
+main {
+  margin: 0;
+  width: 100%;
+  height: auto;
+}
+
+[id='latest'],
+[id='apple-event'],
+[id='funding'],
+[id='main-articles'],
+[id='deals'] {
+  width: 85%;
+  margin: 0 auto;
+}
+
+#apple-event {
+  height: auto;
+  margin-top: 70px;
+}
+
+#funding {
+  height: 300px;
+  margin-top: 50px;
+}
+
+.nav-bar-2 .nav-bar-2-item {
+  margin-top: 15px;
+  margin-left: 10px;
+}
+
+.nav-bar-2 a {
+  color: #aaa;
+  font-family: 'Rajdhani', sans-serif;
+  margin: 0 10px;
+}
+
+#latest {
+  height: auto;
+  margin-top: 70px;
+}
+
+[class*='latest-col'] {
+  width: 24%;
+  margin: 0 0.5%;
+  height: 50%;
+}
+
+.latest-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.pic {
+  position: relative;
+  width: 100%;
+  margin: 20px 0 10px 0;
+  height: auto;
+  top: 0;
+}
+
+section article h3 {
+  font-family: 'Ramabhadra', sans-serif;
+  line-height: 20px;
+  font-size: 0.8em;
+  color: black;
+  margin: 0;
+}
+
+section h1 {
+  margin: 0;
+  font-size: 20px;
+  color: #f42;
+}
+
+.trends-col-9 h2 {
+  display: none;
+}
+
+.deals-col-9 h2 {
+  display: none;
+}
+
+[class*="trends-col"] a h2 {
+  margin: 0;
+  font-size: 20px;
+  color: #f42;
+}
+
+.footer-down div {
+  margin: 0 auto;
+  padding-top: 55px;
+  height: 50px;
+  width: 800px;
```

Comment:
The horizontal scrolling issue you have is happening because you are using pixel units, it is always better to use either `%`, `em` or `rem` for your website to be responsive, try changing the width to `100%` then styling it in the middle of the footer rather than giving it a fixed `800px` width.